.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 23.1.0
+    hooks:
+      - id: black
+        language_version: python3.9
```

Comment:
Related to [this comment](https://github.com/ajaen4/kinesis-flink-hudi-benchmark/pull/3/files#r1132263987). Probably move to 3.8